lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +83,52 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config = json.load(file)
+        return ConfigDTO(
+            seed_urls=config["seed_urls"],
+            total_articles_to_find_and_parse=config["total_articles_to_find_and_parse"],
+            headers=config["headers"],
+            encoding=config["encoding"],
+            timeout=config["timeout"],
+            should_verify_certificate=config["should_verify_certificate"],
+            headless_mode=config["headless_mode"]
+        )
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config_data = json.load(file)
+
+        seed_url = config_data.get("seed_url")
+        if not seed_url or not re.match(r'https?://(www.)?', seed_url):
```

Comment:
is it enough to check that seed urls are OK? maybe I put my lovely nn.ru in the config - you won't complain?

Reply:
Sorry, I didn't quite understand what I should fix

Reply:
we do not work with any website, right? we need seed urls to be exactly of the website we target

Reply:
Fixed